dask/dataframe/io.py

```diff
@@ -462,14 +463,18 @@ def to_hdf(df, path_or_buf, key, mode='a', append=False, complevel=0,
                  "must preserve the order of its input")
 
     # handle lock default based on wether we're writing to a single entity
+    _actual_get = get or _globals.get('get') or get_sync
```

Comment:
Just a clarification: _actual_get does not select the scheduler to use, it tries to predict which scheduler will be used based on configuration. It's only there to change other parameters (like locking) based on the scheduler used.